src/codecs/avif/encoder-meta.ts

```diff
@@ -12,12 +14,14 @@ export const label = 'AVIF';
 export const mimeType = 'image/avif';
 export const extension = 'avif';
 export const defaultOptions: EncodeOptions = {
-  minQuantizer: 16,
-  maxQuantizer: 16,
+  minQuantizer: 33,
+  maxQuantizer: 63,
+  minQuantizerAlpha: 33,
+  maxQuantizerAlpha: 63,
```

Comment:
Lossy alpha by default anyone? I'm undecided if this is the right choice.

Reply:
I’m okay with this.